include/RigidBody/Contacts.h

```diff
@@ -141,21 +141,52 @@ class BIORBD_API Contacts : public RigidBodyDynamics::ConstraintSet
     utils::String contactName(unsigned int i);
 
     ///
-    /// \brief Return the contraints position in the global reference
+    /// \brief Return the normal of the contact of a specified axis
+    /// \param i The axis
+    /// \return The name of the contact normal of a specified axis
+    ///
+    std::vector<bool> rigidContactNormal(unsigned int i);
```

Comment:
What about areAxesNormal ?
The way it is return for now i not a problem for me. But if Vector3 is prefered, just tell me.
I modified the doc.

Reply:
It depends what you do with them. If you need the bool vector for knowing something, then bool are preferred, because otherwise, they will be MX (in casadi), which can't be easily used in a if-else case